lib/src/configuration.dart

```diff
@@ -645,6 +645,8 @@ class SyncError extends RealmError {
         return SyncConnectionError(message, category, SyncConnectionErrorCode.fromInt(code), isFatal: isFatal);
       case SyncErrorCategory.session:
         return SyncSessionError(message, category, SyncSessionErrorCode.fromInt(code), isFatal: isFatal);
+      case SyncErrorCategory.resolve:
+        return SyncResolveError(message, category, SyncResolveErrorCode.values[code]);
```

Comment:
Why not `.fromInt` here`?

Reply:
Because the codes are still not released.